extensions/sql-database-projects/src/test/dialogs/createProjectFromDatabaseDialog.test.ts

```diff
@@ -79,8 +82,10 @@ describe('Create Project From Database Dialog', () => {
 
 	it('Should create default project name correctly when database information is populated', async function (): Promise<void> {
 		sinon.stub(azdata.connection, 'getConnections').resolves([]);
-		sinon.stub(azdata.connection, 'connect').resolves({ connected: true, connectionId: '0', errorMessage: '', errorCode: 0});
+		sinon.stub(azdata.connection, 'connect').resolves({ connected: true, connectionId: '0', errorMessage: '', errorCode: 0 });
 		sinon.stub(azdata.connection, 'listDatabases').resolves(['My Database']);
+		sinon.stub(utils, 'sanitizeStringForFilename').returns('My Database');
+		sinon.stub(newProjectTool, 'defaultProjectNameFromDb').returns('DatabaseProjectMy Database');
```

Comment:
A lot of tests started failing after the data workspace extension * activationEvent was removed because they use some helper functions exposed by data workspace, so those had to updated to stub out those functions.